src/WinRT.Runtime/ApiCompatBaseline.net5.0.txt

```diff
@@ -2,7 +2,10 @@ Compat issues with assembly WinRT.Runtime:
 CannotRemoveAttribute : Attribute 'System.Runtime.CompilerServices.CompilerGeneratedAttribute' exists on 'WinRT.IInspectable.WinRT.IWinRTObject.get_AdditionalTypeData()' in the contract but not the implementation.
 CannotRemoveAttribute : Attribute 'System.Runtime.CompilerServices.CompilerGeneratedAttribute' exists on 'WinRT.IInspectable.WinRT.IWinRTObject.get_QueryInterfaceCache()' in the contract but not the implementation.
 MembersMustExist : Member 'public WinRT.MarshalString.HStringHeader WinRT.MarshalString.HStringHeader WinRT.MarshalString._header' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public System.IntPtr System.IntPtr WinRT.MarshalString._handle' does not exist in the implementation but it does exist in the contract.

```

Comment:
I presume there are no references to these members in generated projections and won't be an API compat breaking change.

Reply:
correct, these members should not have been public to begin with.  our marshaling pattern is constrained to the functional interface (CreateMarshaler, GetAbi, DisposeMarshaler, FromAbi, DisposeAbi, etc)